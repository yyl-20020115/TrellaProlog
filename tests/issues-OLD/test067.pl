:- initialization(main).
:- use_module(library(lists)).

main :-
	prepare(List),
    writeq(List), nl,
    sort(List, ListSorted),
    writeq(ListSorted), nl.

prepare(List) :-
    append([A,B], [B,A], List).
