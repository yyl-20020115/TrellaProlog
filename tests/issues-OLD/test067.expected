[_5,_6,_6,_5]
[_17,_18]
